lib/ae_mdw/db/sync/active_entities.ex

```diff
@@ -30,29 +30,37 @@ defmodule AeMdw.Db.Sync.ActiveEntities do
     with :ok <- result,
          transition when is_tuple(transition) <- Map.get(@transitions, fname),
          true <- valid_args?(fname, args) do
-      exec_transition(state, create_txi, txi, transition)
+      exec_transition(state, create_txi, txi, transition, args)
     else
       _nil_or_false ->
         state
     end
   end
 
-  defp exec_transition(state, create_txi, txi, {:initial, entity_name}) do
+  defp exec_transition(state, create_txi, txi, {:initial, entity_name}, args) do
     m_entity = Model.entity(index: {entity_name, txi, create_txi})
     m_contract_entity = Model.contract_entity(index: {entity_name, create_txi, txi})
+    final_args = args |> Enum.take(2) |> Enum.map(& &1.value)
```

Comment:
Later, it will be possible to match args by positions (for now hardcodes that the first two must match)